build/build-constants.yaml

```diff
@@ -48,10 +48,13 @@
       file-name-prefix: __
 - name: python-versions
   constants:
+    python-build-base-tag: 20190610.2
+    python-runtime-base-tag: 20190610.2
```

Comment:
we don't have a runtime-base for Python, everything comes from the same base (which is currently called "build", admittedly a misleading name"